crates/nu-cli/src/completions/completer.rs

```diff
@@ -829,6 +861,12 @@ impl NuCompleter {
             _ => (),
         }
 
+        // try argument dynamic completion defined by Command first.
```

Comment:
nitpick: I was confused by this at first because I thought this was referring to the other `Command` trait that builtin commands implement. Could you say `nu_plugin::plugin::Command` instead?

Reply:
I thought this feature is meant to be general for all commands, just not yet implemented for builtin ones?

Reply:
My bad I meant to delete this comment, I'd misinterpreted initially